public/locales/en-US.json

```diff
@@ -79,7 +79,9 @@
     "tourDescription": "We'll walk you through each step, and include insight into how Red Hat collects and uses subscription data."
   },
   "curiosity-view": {
-    "openshift": "Red Hat OpenShift",
-    "rhel": "Red Hat Enterprise Linux"
+    "OpenShiftTitle": "Red Hat OpenShift",
+    "OpenShiftSubtitle": "Monitor your Red Hat OpenShift usage by socket or core usage, based on your subscription terms. <0>Learn more about Subscription Watch reporting</0>",
+    "RHELTitle": "Red Hat Enterprise Linux",
+    "RHELSubtitle": "Monitor your Red Hat Enterprise Linux usage by physical, virtual, and public cloud sockets. <0>Learn more about Subscription Watch reporting</0>"
```

Comment:
#### Note: 
Not for this PR, for the future...
Something to iterate on. We should look at removing the casing on `view ids`, forgot about it, apologies